content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -0,0 +1,80 @@
+---
+reviewers:
+- janetkuo
+title: TTL Controller for Finished Resources
+content_template: templates/concept
+weight: ??
+---
+
+{{% capture overview %}}
+
+The TTL controller provides a TTL mechanism to limit the lifetime of resource
+objects that have finished execution. Currently, TTL controller only handles
+[Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) for
+now, and may be expanded to handle other resources that will finish execution,
+such as Pods and custom resources.
+
+Alpha Disclaimer: this feature is currently alpha, and can be enabled with
```

Comment:
in general we avoid words like "currently" which may deprecate soon. Instead we use callouts to label the state of a feature, e.g.
```
{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
```